db/migrate/20160715024414_rename_long_column_in_users.rb

```diff
@@ -4,5 +4,6 @@ def up
   end
 
   def down
+  	rename_column :location_tags, :lon, :long
```

Comment:
You can see this is in the down method of the migration. Without this problems are caused while reversing the migration. An alternative is to use `change` method instead of `up` method in the migration. Will do as you recommend.